specification/cognitiveservices/data-plane/AutoSuggest/readme.nodejs.md

```diff
@@ -6,10 +6,10 @@ Please also specify `--node-sdks-folder=<path to root folder of your azure-sdk-f
 ``` yaml $(nodejs)
 nodejs:
   package-name: azure-cognitiveservices-autosuggest
-  package-version: 1.1.0
+  package-version: 0.1.0-preview
   output-folder: $(node-sdks-folder)/lib/services/autosuggest
   override-client-name: AutoSuggestAPIClient
-  azure-arm: false
+  azure-arm: true
```

Comment:
We don’t usually set this to true for cognitive services, right?

Reply:
Not sure, but I'm definitely going to set it back to false once I have a sample written up.

Reply:
And before I merge this PR.